.github/workflows/ci.yml

```diff
@@ -43,4 +43,5 @@ jobs:
         cmd: install --frozen-lockfile
     - run: chmod +x ./test/run_tests.sh
     - run: yarn test
+    - run: yarn translate:assurance
```

Comment:
It looks like this advice has undone us, because it needs the other docker commands to be run. If you wanted, you could add the docker `start` and `stop` scripts as well. Again, thoughts @mgramigna